L1Trigger/L1TTrackMatch/plugins/L1TrackVertexAssociationProducer.cc

```diff
@@ -148,6 +148,14 @@ class L1TrackVertexAssociationProducer : public edm::global::EDProducer<> {
     std::vector<double> deltaZMaxEtaBounds_;
     std::vector<double> deltaZMax_;
   };
+  struct TTTrackLinkLimitSelector {
+    TTTrackLinkLimitSelector() {}
+    bool operator()(const TTTrackType& t) { return ((t.GTTLinkIndex() < kNTrackSetsTVA) && (t.GTTLinkIndex() >= 0)); }
+
+  private:
+    static const int kNTrackSetsTVA =
+        94;  // corresponds to N_TRACK_SETS_TVA in LibHLS DataFormats/Track/interface/TrackConstants.h
```

Comment:
the comment might be more readable if put above the constant:
```suggestion
    // corresponds to N_TRACK_SETS_TVA in LibHLS DataFormats/Track/interface/TrackConstants.h
    static const int kNTrackSetsTVA = 94;
```

However, why not take it directly from `DataFormats/Track/interface/TrackConstants.h` ?